sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -212,8 +213,16 @@ internal class ViewAnnotationManagerImpl(
         }
       }
     }
-    viewAnnotation.globalLayoutListener = globalLayoutListener
-    inflatedView.viewTreeObserver.addOnGlobalLayoutListener(globalLayoutListener)
+    viewAnnotation.attachStateListener = object : View.OnAttachStateChangeListener {
```

Comment:
I'd add a comment to describe why it's here - not trivial and widely known relation between onViewAttach / onGlobalLayoutListener

Reply:
docs are added inside `ViewAnnotation` class for that field